ui/raidboss/data/05-shb/raid/e6n.txt

```diff
@@ -25,19 +25,22 @@ hideall "--sync--"
 81.7 "Vacuum Slice" sync /:Garuda:4BD5:/
 88.8 "Occluded Front" sync /:Garuda:4BD2:/ window 30,30
 98.3 "Irresistible Pull" sync /:Garuda:4BD6:/
+111.5 "--untargetable--" sync /22:........:Garuda:........:Garuda:00/
 
 # Ifrit solo
-114.0 "Touchdown" sync /:Ifrit:4BE8:/ window 30,30
+114.0 "--targetable--" sync /22:........:Ifrit:........:Ifrit:01/ window 30,30
```

Comment:
This is maybe personal preference, but I wanted to sync on the abilities, but use the 22 lines to generate the targetable/untargetable.  I guess we could sync on them too, but nothing else does.  I think it's worth keeping Touchdown here as well as it does damage.

Reply:
Fair point on Touchdown, that's very true. Is the issue with syncing `22` lines mostly that `test_timeline` currently doesn't support them?

Reply:
Yeah, I think it's that test_timeline doesn't support them mostly.  I also don't know about how reliable they are? The 15 lines are very accurate and don't drift.  14 is a little bit iffy according to an issue folks filed about network vs game log lines.  I think it's also just consistency in terms of syncing to abilities where possible, because that's what everything else uses.